tests/__init__.py

```diff
@@ -53,6 +53,8 @@ def dj_config():
         dj.config.load('./dj_local_conf.json')
     dj.config['safemode'] = False
     dj.config['custom'] = {
+        'ephys_mode': (os.environ.get('EPHYS_MODE')
+                       or dj.config['custom']['ephys_mode']),
```

Comment:
I like this approach to navigating the schema choice. I'll make a note to add documentation highlighting this feature to users